frontend/src/app/coworking/widgets/group-reservation-card/group-reservation-card.widget.html

```diff
@@ -0,0 +1,61 @@
+<!-- group-reservation-card.widget.html -->
+<div style="padding: 8px; box-sizing: border-box">
+  <h2 class="add-users-heading" style="margin: 8px">
+    Add Users to Reservation
+  </h2>
+  <user-lookup
+    style="display: block; margin: 0 8px; box-sizing: border-box"
+    [label]="'Search for users'"
+    (usersChanged)="onUsersChanged($event)">
+  </user-lookup>
+
+  <div style="margin: 8px">
+    <mat-chip-listbox>
+      <mat-chip
+        *ngFor="let user of selectedUsers"
+        [removable]="user.id !== loggedInUser?.id"
+        (removed)="removeUser(user)">
+        {{ user.first_name }} {{ user.last_name }}
+        <mat-icon matChipRemove *ngIf="user.id !== loggedInUser?.id"
+          >cancel</mat-icon
+        >
+      </mat-chip>
+    </mat-chip-listbox>
+  </div>
+
+  <!-- Button container -->
+  <div
+    style="
+      display: flex;
+      justify-content: space-between;
+      margin: 12px 8px 45px;
+      box-sizing: border-box;
+    ">
+    <!-- Add Users Button -->
+    <button
+      mat-button
+      color="primary"
```

Comment:
Use the `outlined` style for buttons! Also, these buttons can be bottom-aligned

Reply:
Changed to a mat-stroked-button and bottom aligned the buttons to the bottom of the dialog box. However, I'd argue that having the buttons closer to the selected members section looks better (if this is what you meant by bottom align)! 

Snippets:

Without bottom aligning (the box has to be this big to prevent overflow when selecting multiple members):
![image](https://github.com/unc-csxl/csxl.unc.edu/assets/107937474/38f8e160-ddc3-486c-93dd-058d6b5bbbb2)

With bottom aligning:
![image](https://github.com/unc-csxl/csxl.unc.edu/assets/107937474/f259ae9a-cf98-4438-8c6e-e017ad35d81c)